frontends/pytorch/test/acap_export/test_export_add3.py

```diff
@@ -13,6 +13,9 @@
 
 mb = torch_mlir.ModuleBuilder()
 with mb.capture_function("add3", [t0, t1, t2]) as f:
+  # t0 = f.import_tensor(t0)
```

Comment:
what are these comments for?

Reply:
Oops - I thought I had backed out all of the hacking I had done to find the solution, but had forgotten about import_tensor. Just removed it entirely.